### PR TITLE
(CDAP-4045) Optionally disable Kafka in AM.

### PR DESCRIPTION
**Pull request by chtyim**
- This is a workaround for MapR.

### CODE REVIEW

File: cdap-common/src/main/resources/cdap-default.xml [hunk 1]
```diff
@@ -806,6 +806,14 @@
   </property>
 
   <property>
+    <name>master.collect.containers.log</name>
+    <value>true</value>
+    <description>
+      Collection of all containers log into master process log.
```
Comment (awholegunch) [inline cdap-common/src/main/resources/cdap-default.xml]: Does this mean:
```
Determines if the master should collect all container logs into a single master process log

Comment (dereklwood) [inline cdap-common/src/main/resources/cdap-default.xml]: I would suggest something like "Determines if container logs are streamed back to the cdap-master process log"

Comment (chtyim) [inline cdap-common/src/main/resources/cdap-default.xml]: fixed.

**Comment by chtyim** [comment]
Build passed: http://builds.cask.co/browse/CDAP-DUT3135-3

**Comment by sreevatsanraman** [comment]
LGTM :+1: Build is green merging to run the integration tests.

**Pull request merged**